requirements/default.txt

```diff
@@ -720,6 +732,12 @@ markupsafe==2.0.1 \
     # via
     #   -r requirements/default.in
     #   jinja2
+moz-l10n[xml]==0.4.0 \
```

Comment:
Note that running `make requirements` will change this to `moz-l10n`. 
See: https://github.com/astral-sh/uv/issues/1595

Do we want/need to add `--no-strip-extras` to the compile command?